newIDE/app/src/MainFrame/TabsTitlebar.js

```diff
@@ -53,6 +205,39 @@ export default function TabsTitlebar({
     [backgroundColor]
   );
 
+  const onHoverEditorTab = React.useCallback(
+    (editorTab: ?EditorTab) => {
+      if (isTouchscreen) {
+        setTooltipData(null);
+        return;
+      }
+
+      if (tooltipTimeoutId.current) {
+        clearTimeout(tooltipTimeoutId.current);

```

Comment:
should that timeout be cleared when the component unmounts? (it shouldn't unmount technically, but...)

Reply:
Good idea